### DISCUSSION TITLE
Error when setting custom instructions: The additional field system conflicts with the existing field text

### DISCUSSION THREAD

**Discussion by EmreErdogan** [danny-avila/LibreChat#4470] [Troubleshooting]
**Description:**
When attempting to submit a message with custom instructions set for AWS Bedrock's `anthropic.claude-3-5-sonnet-20240620-v1:0` model, an error occurs indicating a conflict between the 'system' and existing field text. This prevents the use of custom instructions and needs to be resolved.

**Steps to Reproduce:**
1. Set custom instructions
2. Submit a message to the assistant

**Expected Behavior:**
The AI assistant should work when custom instructions are set successfully.

**Actual Behavior:**
The error in the following screenshot is shown on the screen.

<img width="823" alt="Screenshot 2024-10-20 at 11 50 07" src="https://github.com/user-attachments/assets/2b21eb4c-037e-4516-884d-32ea9d07bd30">

**Container logs:**

> 2024-10-20 07:22:05 error: [handleAbortError] AI response error; aborting request: The additional field system conflicts with the existing field text. Remove system and try again.
> 2024-10-20 07:22:46 error: [api/server/controllers/agents/client.js #sendCompletion] Unhandled error type The additional field system conflicts with the existing field text. Remove system and try again.

**Environment:**
- LibreChat Version: v0.7.5-rc2
- LLM Version: `anthropic.claude-3-5-sonnet-20240620-v1:0`
- Hosting: HuggingFace

**Related Issues:**
- https://github.com/danny-avila/LibreChat/issues/2620#issuecomment-2364780848

**Comment by danny-avila** [comment]
Maybe this is fixed already, if the commit head is at `v0.7.5-rc2` for you and not latest from main branch.

What do the parameters on the right show for you?

![image](https://github.com/user-attachments/assets/9c98b02e-39da-4093-95fa-ef374b9c6da0)

**Reply by EmreErdogan** [reply]
Here's a screenshot showing the parameters.

<img width="1198" alt="Screenshot 2024-10-20 at 14 40 20" src="https://github.com/user-attachments/assets/3d172c26-2035-459e-ad1c-ea70d7d965a4">

**Reply by danny-avila** [reply]
I'm sure this was fixed as I can't reproduce it. I will update the latest stable release soon then report back, to see if you can still reproduce it after updating.

**Reply by EmreErdogan** [reply]
I've just updated the installation by doing a "Factory rebuild" and I no longer get any errors 👌 
But the custom instructions seem to have no effect on the generated outputs though 🤔 It might be a shortcoming of the model.

**Reply by EmreErdogan** [reply]
Uh oh! Received the same error again.

> 2024-10-24 10:41:40 error: [api/server/controllers/agents/client.js #sendCompletion] Unhandled error type The additional field system conflicts with the existing field text. Remove system and try again.
2024-10-24 10:41:40 error: [handleAbortError] AI response error; aborting request: The additional field system conflicts with the existing field text. Remove system and try again.

**Reply by danny-avila** [reply]
Are you switching models when this happens?

**Reply by EmreErdogan** [reply]
No. The same model 🤷

**Reply by oaustegard** [reply]
We're seeing the same whenever we use a Preset with a systems message

```
{"$fault":"client","$metadata":{"attempts":1,"httpStatusCode":400,"requestId":"e4139d8f-75ce-444d-a831-32f39af607fe","totalRetryDelay":0},"level":"error","message":"[api/server/controllers/agents/client.js #sendCompletion]
ValidationException: The additional field system conflicts with the existing field text. Remove system and try again.
    at de_ValidationExceptionRes (/app/node_modules/@aws-sdk/client-bedrock-runtime/dist-cjs/index.js:1196:21)
    at de_CommandError (/app/node_modules/@aws-sdk/client-bedrock-runtime/dist-cjs/index.js:1029:19)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async /app/node_modules/@smithy/middleware-serde/dist-cjs/index.js:35:20
    at async /app/node_modules/@smithy/core/dist-cjs/index.js:165:18
    at async /app/node_modules/@smithy/middleware-retry/dist-cjs/index.js:320:38
    at async /app/node_modules/@aws-sdk/middleware-logger/dist-cjs/index.js:34:22
    at async ChatBedrockConverse._streamResponseChunks (/app/node_modules/@langchain/aws/dist/chat_models.cjs:693:26)
    at async ChatBedrockConverse._generateUncached (/app/node_modules/@langchain/aws/node_modules/@langchain/core/dist/language_models/chat_models.cjs:157:34)
    at async ChatBedrockConverse.invoke (/app/node_modules/@langchain/aws/node_modules/@langchain/core/dist/language_models/chat_models.cjs:60:24)  
```
"name": "LibreChat", "version": "v0.7.5",

based on package.json

**Comment by ManuelRamoss** [comment]
Hi, I am facing the same issue and the logs are the following.
![Screenshot 2024-10-21 at 10 53 12 a m](https://github.com/user-attachments/assets/6fa850f9-6af5-4ae8-a97f-10eb56b4e2ca)

**Reply by danny-avila** [reply]
This is not the same issue, it seems your AWS credentials are invalid.
